dipy/denoise/localpca.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+import scipy as sp
+
+def localpca(arr, sigma, patch_radius=1, tou=0, rician=True):
+    '''
```

Comment:
default tou should be None. Below when checking tou you should use:
if tou is None: